gruntfile.js

```diff
@@ -537,4 +439,11 @@ module.exports = function (grunt) {
       }
     }
   });
+
+  grunt.registerTask('vendor', 'vendor:<min|reg>', function(min) {
```

Comment:
The mapping is `'src.'+type+'Vendor'` <-> `v[type][min]`, where `v` <-> `vendor.yml`. What additional documentation do you expect?